src/Microsoft.Extensions.OData.Migration/ExtensionMethods.cs

```diff
@@ -0,0 +1,143 @@
+﻿using Microsoft.OData.Edm;
+using System;
+using System.Globalization;
+
+namespace Microsoft.Extensions.OData.Migration
+{
+    /// <summary>
+    /// Contains extension methods for V3 IEdmType and System.Uri
+    /// </summary>
+    public static class ExtensionMethods
+    {
+        /// <summary>
+        /// (probably temporary until cleaner solution) method to concatenate a string to URI
+        /// </summary>
+        /// <param name="uri">base URI</param>
+        /// <param name="extra">string to append</param>
+        /// <returns>base URI with string appended</returns>
+        public static Uri Append(this Uri uri, string extra)

```

Comment:
Extentensions.cs is too generic. Maybe break this up into multiple classes targeting different types. UriExtensions.cs, EdmExtensions.cs